frontend/app/src/pages/docs/embedding-model.mdx

```diff
@@ -90,6 +90,30 @@ Cohere provides industry-leading large language models (LLMs) and RAG capabiliti
 
 For more information, see the [Cohere Embed documentation](https://docs.cohere.com/docs/cohere-embed).
 
+### Amazon Bedrock
+
+Amazon Bedrock is a fully managed foundation models service that provides a range of large language models and embedding models.
+
+**Supported Models**:
+
+| Embedding Model                 | Vector Dimensions | Max Tokens |
+| ------------------------------- | ----------------- | ---------- |
+| `amazon.titan-embed-text-v2:0`  | 1024              | 8192       |
+| `amazon.titan-embed-text-v1`    | 1536              | 8192       |
+| `amazon.titan-embed-g1-text-02` | 1536              | 8192       |
```

Comment:
It seems AWS provides more models than these three?

For example cohere:xxxxx

Reply:
I suggest to document featured models and add a *view more* link to provider's document page.

Reply:
Done!